common/src/IO/Path.cpp

```diff
@@ -409,6 +410,28 @@ namespace TrenchBroom {
             return *this + relativePath;
         }
 
+        Path Path::makeRelative() const {
+            if (isEmpty()) {
+                throw PathException("Cannot make relative path from an empty reference path");
+            }
+
+            if (!isAbsolute()) {
+                throw PathException("Cannot make relative path from relative reference path");
+            }
+
+#ifdef _WIN32
```

Comment:
Path's OS-specific behaviour is confusing and I don't really trust it. Not sure how this works with Windows style absolute paths on UNIX systems, or vice versa, but I guess the test cases must be testing these?

Reply:
Actually I don't think they do. So if you try to parse a Windows-style path on macOS, it will fail, and vice versa. We can consider unifying the platform dependent behavior.